samples/snippets/fsharp/azure/table-storage.fsx

```diff
@@ -0,0 +1,193 @@
+open System
+open System.IO
+open Microsoft.Azure // Namespace for CloudConfigurationManager
+open Microsoft.WindowsAzure.Storage // Namespace for CloudStorageAccount
+open Microsoft.WindowsAzure.Storage.Table // Namespace for Table storage types
+
+//
+// Get your connection string.
+//
+
+let storageConnString = "..." // fill this in from your storage account
+(*
+// Parse the connection string and return a reference to the storage account.
+let storageConnString = 
+    CloudConfigurationManager.GetSetting("StorageConnectionString")
+*)
+//
+// Parse the connection string.
+//
+
+// Parse the connection string and return a reference to the storage account.
+let storageAccount = CloudStorageAccount.Parse(storageConnString)
+
+//
+// Create the Table service client.
+//
+
+// Create the table client.
+let tableClient = storageAccount.CreateCloudTableClient()
+
+//
+// Create a table.
+//
+
+// Retrieve a reference to the table.
+let table = tableClient.GetTableReference("people")
+
+// Create the table if it doesn't exist.
+table.CreateIfNotExists()
+
+//
+// Add an entity to a table. The last name is used as a partition key.
+//
+
+type Customer(firstName, lastName, email: string, phone: string) =
+    inherit TableEntity(partitionKey=lastName, rowKey=firstName)
+    new() = Customer(null, null, null, null)
+    member val Email = email with get, set
+    member val PhoneNumber = phone with get, set
+
+let customer = 
+    Customer("Larry", "Boomer", "larry@example.com", "425-555-0101")
```

Comment:
Same as previous question re. names, does this name come from our pre-approved list?